.github/workflows/standard-its.yml

```diff
@@ -94,7 +94,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jdk: [8, 17, 21]
+        jdk: ['8', '17', '21.0.4']
```

Comment:
May want to update this check as well with the more specific `21.0.4` JDK version - https://github.com/apache/druid/blob/master/integration-tests/script/docker_build_containers.sh#L29

Reply:
I've unpatched this file for now - wdyt?

Reply:
Sounds OK to me.

Reply:
Sounds reasonable to me